src/mars_patcher/data/schema.json

```diff
@@ -27,6 +27,11 @@
                             },
                             "Item": {
                                 "$ref": "#/$defs/ValidItems"
+                            },
+                            "ItemMessage": {
+                                "type": "string",
+                                "description": "Specifies what text should appear on acquiring this item. Two lines are available, with an average of 28 characters per line. Use \n to force a line break",
```

Comment:
mention that if not provided, that a message based on `Item` will be shown instead for clarity.